Header header 
State[] ref_state
float32[] goal_pose
bool goal_reached